src/Progress.UnitTests/ProgessController/VsOutputWindowPaneNotifierTests.cs

```diff
@@ -0,0 +1,155 @@
+//-----------------------------------------------------------------------
+// <copyright file="VsOutputWindowPaneNotifierTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.Progress.Controller.ErrorNotification;
+using System;
+
+namespace SonarLint.VisualStudio.Progress.UnitTests
+{
+    /// <summary>
+    /// Tests for <see cref="VsOutputWindowPaneNotifier"/>
+    /// </summary>
+    [TestClass]
+    public class VsOutputWindowPaneNotifierTests
+    {
+        private ConfigurableServiceProvider serviceProvider;
+        private StubVsUIShell uiSHell;
+        private VsOutputWindowPaneNotifier testSubject;
+        private StubVsOutputWindowPane outputWindowPane;
+        private Exception generatedException;
```

Comment:
Does this variable need to be in a field? The value doesn't seem to be checked anywhere. It's also only initialised if _Setup_ is called, and the tests that use it have to know that. It would be safer and more maintainable to have a lazily-initialised property if the variable really needs to be a field, although it looks as if a local variable would do.